lib/archethic/beacon_chain/subset.ex

```diff
@@ -354,7 +356,13 @@ defmodule Archethic.BeaconChain.Subset do
   end
 
   defp handle_summary(time, subset) do
-    beacon_slots = SummaryCache.pop_slots(subset)
+    beacon_slots =
+      subset
+      |> SummaryCache.stream_current_slots()
```

Comment:
This function is now returning a Stream, so the next Logger does not write what it should